src/shared/vars.js

```diff
@@ -1,4 +1,6 @@
-const VERSION = '__skate_0_14_0';
```

Comment:
Just added that to emphasise that this has been broken since 0.14.0 - no action needed :)